### PR TITLE
chore: Tiny adjustements to eslint

### PR DESCRIPTION
**Pull request by vhf**

### CODE REVIEW

File: static/.eslintrc.json [hunk 1]
```diff
@@ -60,23 +60,24 @@ Do not edit this file in your local project
...
-    "new-cap": [2, {"newIsCap": true, "capIsNew": false }],
+    "new-cap": [2, {"newIsCap": true, "capIsNew": false}],
     "new-parens": 2,
-    "newline-per-chained-call": [2, {"ignoreChainWithDepth": 4 }],
+    "newline-per-chained-call": [2, {"ignoreChainWithDepth": 4}],
     "no-array-constructor": 2,
     "no-caller": 2,
     "no-class-assign": 2,
     "no-cond-assign": 2,
+    "no-console": [1, {"allow": ["error"]}],
```
Comment (vhf) [inline static/.eslintrc.json:78]: @marcbachmann

File: static/.eslintrc.json [hunk 2]
```diff
@@ -60,23 +60,24 @@ Do not edit this file in your local project
...
-    "key-spacing": [2, {"beforeColon": false, "afterColon": true }],
-    "keyword-spacing": [2, {"before": true, "after": true }],
+    "key-spacing": [2, {"beforeColon": false, "afterColon": true}],
+    "keyword-spacing": [2, {"before": true, "after": true}],
     "linebreak-style": [2, "unix"],
     "max-len": ["error", {
       "code": 100,
       "ignoreRegExpLiterals": true,
-      "ignorePattern": "^\\s*const\\s.+=\\s*require\\s*\\(/"
+      "ignorePattern": "^\\s*const\\s.+=\\s*require\\s*\\("
```
Comment (vhf) [inline static/.eslintrc.json:69]: Pattern wasn't good, required a `/` after `(` so only worked on

`const foo = require(/`

**Comment by vhf** [comment]
I commented the two modifications, see above. Other things are whitespaces.

https://github.com/upfrontIO/apply-guides/pull/29/files?w=1

File: static/.eslintrc.json (hunk 1 above)
Comment (marcbachmann) [reply static/.eslintrc.json:78]: Nice. Thanks.

**Review by marcbachmann — APPROVED** [review]

**Pull request merged**